### PR TITLE
Possible fix for linked images

### LINKED ISSUE #164: Linked image in footer widgets has underline.
https://cloudup.com/ijzlfdhLGGf

I'm using the Monster widget in the footer widget area.

### PR DESCRIPTION
**Pull request by joshcummingsdesign**
Possible fix for linked images for content and widget areas. Would close #164

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -909,6 +909,14 @@ a:hover .nav-title,
 	color: #fff;
 }
 
+/* Fixes linked images */
+.entry-content a img,
+.widget a img {
+	background: #fff;
+	box-shadow: 0 0 0 6px #fff;
+	filter: progid:DXImageTransform.Microsoft.DropShadow(OffX=0, OffY=5, Color=#ffffff);
```
Comment (laurelfulford) [inline style.css]: What do you think about moving this to the ie8.css stylesheet, since it's only needed for that version of IE?

File: style.css [hunk 2]
```diff
@@ -3282,4 +3290,4 @@ article.panel-placeholder {
 	.twentyseventeen-front-page .entry-content blockquote.alignright {
 		margin-right: -20%;
 	}
-}
\ No newline at end of file
+}
```
Comment (laurelfulford) [inline style.css:3292]: This is probably coming from your editor, but can you please re-add the newline at the bottom of the file?

**Comment by laurelfulford** [comment]
Thanks for putting this all together, @joshcummingsdesign!

I added a couple super-minor comments on the code - just let me know if you have any questions!

File: style.css [hunk 3]
```diff
@@ -580,7 +580,7 @@ q:after {
 
 /* Typography for Thai Font */
 
-html[lang="th"] h1, 
+html[lang="th"] h1,
```
Comment (laurelfulford) [inline style.css:583]: Unrelated to the fix, but that's okay - it looks like your editor cleaned up a stray extra space 🙂

**Comment by joshcummingsdesign** [comment]
Moved the filter rule to ie8.css and addressed spacing issues.

**Comment by laurelfulford** [comment]
Thanks @joshcummingsdesign!

**Pull request merged**